src/mitol/google_sheets/sheet_handler_api.py

```diff
@@ -31,6 +40,23 @@ class SheetHandler:
     spreadsheet = None
     sheet_metadata = None
 
+    def is_configured(self):
+        """
+        Checks for required settings.
+
+        Returns:
+            bool: false if required settings are missing
+        """
+        missing = []
+        for variable in REQUIRED_GOOGLE_SHEETS_SETTINGS:
+            try:
+                if not getattr(settings, variable):
+                    missing.append(variable)
+            except AttributeError:
+                missing.append(variable)
+                log.exception(f"{variable} is not set.")
```

Comment:
The body of the `if` statement here is unreachable since a default value isn't provided to `getattr`. The options to clean this up are:
- Keep the exception handling and call `getattr(settings, variable)` while ignoring the return value.
- Remove the exception handling and use `if getattr(settings, variable, None) is None:`